tox.ini

```
@@ -9,6 +9,7 @@ testpaths = credentials/apps
 [testenv]
 deps =
     django42: -r requirements/django.txt
+    django52: -r requirements/django.txt
```

This isn't actually testing Django 5.2, the django.txt file only pins to the current version of django.
```suggestion
    django52: django>=5.2
```